akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -753,27 +754,39 @@ final class Partition[T](val outputPorts: Int, val partitioner: T ⇒ Int, val e
   override val shape: UniformFanOutShape[T, T] = UniformFanOutShape[T, T](in, out: _*)
```

Comment:
As mentioned during hackathon, this needs to extend the documentation with information that `* Adheres to the [[ActorAttributes.SupervisionStrategy]] attribute.` (exact that wording), in the docs on the class ` final class Partition[T]`